opal/mca/threads/argobots/threads_argobots_wait_sync.c

```diff
@@ -21,6 +21,32 @@
 static opal_mutex_t wait_sync_lock = OPAL_MUTEX_STATIC_INIT;
 static ompi_wait_sync_t *wait_sync_list = NULL;
 
+void wait_sync_global_wakeup_st(int status)
+{
+    ompi_wait_sync_t *sync;
+    for (sync = wait_sync_list; sync != NULL; sync = sync->next) {
+        wait_sync_update(sync, 0, status);
+    }
+}
+
+void wait_sync_global_wakeup_mt(int status)
```

Comment:
This code seems identical to the code in `pthreads/threads_pthreads_wait_sync.c`. Is it possible to move this code into a common subdirectory?

Reply:
Thanks. Yes, we should, at least, for Argobots and Qthreads, but I also hesitate to change the Pthreads code too much. To be honest, I need someone who pushes me to change the design that can affect Pthreads. I will write my idea about this in this PR's main GitHub conversation thread.